Update CentOS7 image

rzikm opened the pull request:
Contributes to #69664.

This new image should have updated (non-expired) CA certificates.

rzikm commented:
/azp run runtime-libraries-coreclr outerloop

ViktorHofer commented:
Before merging please make sure that the test failures aren't related.

rzikm commented:
I am getting "A call to an Azure DevOps api returned 401, which may indicate a bad 'System.AccessToken' value." on all outerloop CI legs, any idea what is going on, @ViktorHofer?

ViktorHofer commented:
Nope, but maybe one of the known issues that were recently communicated by core-eng? cc @dotnet/dnceng

riarenas commented:
It looks like this pipeline "runtime-libraries-coreclr outerloop" still hasn't re-enabled the use of secrets from forks after @agocke disabled them.

riarenas commented:
I have re-enabled the setting.

rzikm commented:
/azp run runtime-libraries-coreclr outerloop

rzikm commented:
There was a wrong image tag (same bug which caused https://github.com/dotnet/runtime/issues/70005), rerunning

rzikm commented:
/azp run runtime-libraries-coreclr outerloop

rzikm committed:
Update CentOS7 image

This updates the CentOS7 image used for libraries CI run to the newest version, which includes updates for `ca-certificates` package which in turn fixes certificate validation issues during test runs.

rzikm commented:
/azp run runtime-libraries-coreclr outerloop

rzikm commented:
Test failures are unrelated (none of them are on CentOS7).
